addons/l10n_bg/data/account_tax_template_data.xml

```diff
@@ -981,6 +989,7 @@
                     'repartition_type': 'tax',
                 }),
             ]"/>
+        <field name="active" eval="False"/>
     </record>
     <!-- Purchase [END] -->
-</odoo>
+</odoo>
```

Comment:
missing EOF creating diff but no care because this file disappear in the next version